irida_staramr_results/cli.py

```diff
@@ -47,7 +47,7 @@ def main():
                      "Test123!")
 
     try:
-        analyses = irida.get_analyses_from_projects(args_dict["project"])
+        amr_analyses = irida.get_amr_analysis_results(args_dict["project"])
```

Comment:
This function can throw a `IridaKeyError` so we need to have an `except IridaKeyError as e:` as part of this `try` to catch it.
With the other changes to `_get_link` usage I am recommending for this PR, it will make `HTTPError`'s not occur, so they could be removed from the `try/except`

Reply:
Done, I added `except IridaKeyError as e:`. I also found that my error messages were misleading (saying analysis not found instead of project id not found). I fixed that as well.